src/test/resources/dlsfls/roles_mapping_tlq.yml

```diff
@@ -0,0 +1,7 @@
+---
+_meta:
+  type: "rolesmapping"
+  config_version: 2
+
+sg_dls_tlq_lookup:
```

Comment:
`sg` is probably wrong here (same for the role)

Reply:
changed naming